src/Applications/GEOSdas_App/fvsetup

```diff
@@ -10031,6 +10046,8 @@ sub init_agcm_pert_rc {
 sub copy_resources {
   my ($chemrc, $ensFLG);
   my ($flags, $label, $string, $vars, $cmd);
+  my ($fname, $fname_old, $expid_old, $target);
+  my ($lfo_tarfile, @lfo_files, $target, $expid_old, $fname, $fname_old);
```

Comment:
I recognize `lfo_tarfile` is just a variable name, but for consistency it may be better to use `bkglfo_tarfile`

Reply:
same as above